### ISSUE TITLE
Not working on Windows 10 Universal Platform

### ISSUE THREAD

**Issue by andrewwebber** [Microsoft/code-push#306]
I have taking an identical example and was successful in iOS but not on React Native Windows on a Lumia 640XLDS

When an update is detected is seems to hang when trying to boot the new bundle and as the loading time is so long I think windows aborts the app starting. It is possible to get into a state where after an update has been restarted that a rollback does not take place

```javascript
import React, { Component } from 'react';
import {
  AppRegistry,
  StyleSheet,
  Text,
  View
} from 'react-native';
import codePush from "react-native-code-push";

class examplepush extends Component {
  constructor(){
    super();
  }
  
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>
          Welcome to React Native!
        </Text>
        <Text style={styles.instructions}>
          To get started, edit index.windows.js
        </Text>
        <Text style={styles.instructions}>
          Press Ctrl+R to reload,{'\n'}
          Shift+F10 or shake for dev menu
        </Text>
        <Text>Version 1.0.15</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },
});

let codePushOptions = { checkFrequency: codePush.CheckFrequency.ON_APP_RESUME };
App = codePush(codePushOptions)(examplepush)

AppRegistry.registerComponent('examplepush', () => App);

```

**Comment by silhouettes** [comment]
Hi @andrewwebber, sorry for the slow response. Just so that you're not left hanging, we're currently beyond swamped with all of the support requests, so it may be an extremely long time before we're able to get around to this. If anyone from the community would like to help out, we would love to have any contributions you can make!

**Comment by rozele** [comment]
I think this is resolved with https://github.com/Microsoft/react-native-code-push/pull/545

**Comment by silhouettes** [comment]
@andrewwebber - this is now available in `react-native-code-push@1.15.0-beta`. I'm going to close this for now, but please let me know if this solves it for you!

**Comment by andrewwebber** [comment]
@silhouettes @rozele I can now confirm, as a result of the fix, this is now working on UWP. Much appreciated!
